jupyter-notebooks/orders/ordering_and_delivery.ipynb

```diff
@@ -13,7 +13,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```

Comment:
First link is a 404 😆 - I recommend checking all the links

This notebook imports from `planet` but doesn't seem to use it? It looks like it's using requests. But using `planet` for auth? I'd drop the use of planet for auth, it won't affect requests.

Maybe we should specify somewhere which notebooks are using the client and which are using requests? I'm of a mind to remove requests notebooks for functionality that is in the client, curious what you think!

For now, I'll end my review here since I don't see any SDK code in this notebook.